liquibase-core/src/main/java/liquibase/ui/interactive/getter/DirectoryGetter.java

```diff
@@ -0,0 +1,29 @@
+package liquibase.ui.interactive.getter;
+
+import liquibase.ui.interactive.AbstractCommandLineValueGetter;
+
+import java.io.File;
+
+public class DirectoryGetter extends AbstractCommandLineValueGetter<String> {
```

Comment:
Since we are plugging the interactivity into the ConfigurationValueProvider system, why isn't this using the ConfigurationValueConverter structures we already have?